core/lib/src/shield/policy.rs

```diff
@@ -286,12 +286,24 @@ pub enum Hsts {
 
     /// Google maintains an [HSTS preload service] that can be used to prevent
     /// the browser from ever connecting to your site over an insecure
-    /// connection. Read more at [MDN]. Don't enable this before you have
-    /// registered your site.
+    /// connection. Read more at [MDN]. This will need to be enabled before
+    /// registering your site.
+    ///
+    /// A flag is available to include (iff true) the `includeSubDomains`
```

Comment:
There is a typo

Reply:
That iff was meant to be short for if and only if.